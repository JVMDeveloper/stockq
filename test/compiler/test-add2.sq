print(5.5 + 4);
print(2 + 3.1);
