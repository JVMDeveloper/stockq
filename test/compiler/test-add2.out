9.500000
5.100000

